content/blog/2017/2017-04-11-JW-agenda-announce.adoc

```diff
@@ -0,0 +1,55 @@
+---
+layout: post
+title: "Jenkins World 2017 Agenda is Live"
+tags:
+- event
+- jenkinsworld
+author: alyssat
+---
+We are excited to announce the agenda for link:https://www.cloudbees.com/jenkinsworld/home[Jenkins World 2017]. This year’s event promises to have something for everyone - whether you are a novice, intermediate or advanced user...we’ve got you covered.  
```

Comment:
Definitely needs a "this is a guest post from CloudBees" thing at the start, to make clear who "we" is throughout the post.